api/reference/1.0/notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+---
+
+# Notes
+
+`Notes` are textual annotations you can add to Launch resources of these types:
+- `DataElements`
+- `RuleComponents`
+- `Rules`
+- `Extensions`
+- `Libraries`
+- `Properties`
+
+Those six types are collectively known as the `Notable` resources.
+
+When a `Notable` resource is deleted, its associated `Notes` are also deleted.
+
+## `Notes` are Comments
+Launch doesn't attach any meaning to `Notes`.
+Adding a `Note` won't change the behavior of your `Property`.
+
+You might use `Notes` to:
+- provide background information
+- function as to-do lists
+- pass along resource usage advice
+- give instructions to other team members
+- record historical context
+- remind yourself why this resource is built the way it is
+
+But Launch ignores `Notes` so you are free to use them any way you want.
+
+## Revisable Notables
+
+In the case of revisable types
+(`DataElements`, `RuleComponents`, `Rules`, and `Libraries`),
+any `Notes` must be created on the current (`head`)
+revision. They may not be attached to other revisions.
+
+However, `Notes` may be _read_ from revisions. In such cases, the endpoints
+will return only the `Notes` that existed before the creation of the revision.
```

Comment:
@thebenrob I can see a potential need for adding notes to an old revision. Perhaps you forgot to document a change when it was made last week and now you want to document it because you are handing off some work to another person. Do we care to support this type of use case?

Reply:
The current feature set was modeled after DTM in the name of "feature parity".  I think you may be right that some will ask for this, but it raises a number of complications that would have to be dealt with and I'm willing to wait and see what the users think.

Reply:
I think not right now.  The current set of functionality was done in the name of "feature parity" with DTM.  I understand the use case and I think you could be right that some will want it, but it would introduce a number of complexities that we didn't want to deal with in the first iteration.  One example, if you try to add a note to head, do we attach it to head and then later attach it to the next revision?  Do we just cut a revision for you and put it there?  If Notes are on revisions, how would I see all notes for any revision?
In any case, I consider this a releasable chunk and we can iterate later based on user feedback.

Reply:
👍